sites/docs/docs/getting-started/install-evidence.md

```diff
@@ -7,6 +7,18 @@ hide_table_of_contents: false
 
 If this is your first time doing something like this, or you just need a reminder, check out the [Detailed Installation Walkthrough.](/walkthroughs/installation)
 
+
```

Comment:
FYI : Flipped the order of ## System Requirements  and ## Install & Launch and added a section for using Docker instead of NPM with a link.